src/book_prep.py

```
@@ -542,18 +532,16 @@ def _is_mam_host(url_or_host: str) -> bool:
         try:
             from src.openlibrary import openlibrary_manager
 
-            openlibrary_data = await openlibrary_manager.search_by_work_id(openlibrary_id, base_dir=base_dir, debug=meta.debug)
+            openlibrary_data = await openlibrary_manager.search_by_work_id(openlibrary_id, base_dir=base_dir)
         except Exception as ex:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)